### PR TITLE
add missing libX dependencies for cairo

### PR DESCRIPTION
**Pull request by boegel**
When looking into adding support for Glade, I ran into missing dependencies for cairo~~, and some misspecified build deps in X11 libraries that should actually be runtime dependencies (as specified in the pkg-config files for these libraries).~~

~~The latter was changed in the latest `2016` easyconfigs for the X11 libraries done by @hajgato (cfr. #2549, #2563, #2614)~~

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2151.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7b330d528ba95ff5aeae for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2668.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4164b01764d2d23be203 for a full test report.

**Comment by hajgato** [comment]
`protos` are always headers and nothing else, so I suggest them to use only as builddeps, and never deps. This means that if you need to build something that requires some X11 libs, you might have to include some of the `protos` as a builddep as well. (not always).

**Comment by hajgato** [comment]
@boegel: mayve we have to explicitly add `libX11` as dependency (despite is automatically loaded by those Xlibs). The shared object is also depends on libX11.so. But it is up to your taste, otherwise it is fine.

**Comment by boegel** [comment]
@hajgato: makes sense, will do, thanks for the suggestion

**Comment by hajgato** [comment]
@boegel:
shoot, you might need to add 
```python
builddependencies = [
    ('renderproto', '0.11'),
]
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2151.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/6218860aae17898d01c2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2668.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/bd2ac6ff7c66a35c7908 for a full test report.

**Comment by boegel** [comment]
@hajgato: ok, will add that too

File: easybuild/easyconfigs/c/cairo/cairo-1.14.6-intel-2016a.eb [hunk 1]
```diff
@@ -24,6 +24,13 @@ dependencies = [
     ('fontconfig', '2.11.94'),
     ('expat', '2.1.0'),
     ('GLib', glibver),
+    ('libX11', '1.6.3'),
+    ('libxcb', '1.11.1'),
+    ('libXrender', '0.9.9'),
+    ('libXext', '1.3.3'),
+]
+builddependencies = [
```
Comment (hajgato) [inline easybuild/easyconfigs/c/cairo/cairo-1.14.6-intel-2016a.eb]: above dependencies?
extra empty line?

Comment (boegel) [inline easybuild/easyconfigs/c/cairo/cairo-1.14.6-intel-2016a.eb]: We don't have strict guidelines for this imho..

I think you'll find all combos w.r.t. order and empty line in between or not.

Let's change this consistently once we have an automated check for it.

Comment (hajgato) [inline easybuild/easyconfigs/c/cairo/cairo-1.14.6-intel-2016a.eb]: OK, then. :)

**Comment by boegel** [comment]
Thanks for the review @hajgato!

**Pull request merged**